sunbeam-python/sunbeam/provider/local/commands.py

```diff
@@ -481,6 +481,17 @@ def bootstrap(
                 proxy_settings=proxy_settings,
             )
         )
+        plan4.append(
+            DeployMicrocephApplicationStep(
```

Comment:
Can we add a comment about why this is needed ?

This is not immediately obvious that you want to fill in the relation offers

Reply:
Done